UnitTests/CardPaymentsTests/ConfirmPaymentSourceRequest_Tests.swift

```diff
@@ -73,4 +73,39 @@ class ConfirmPaymentSourceRequest_Tests: XCTestCase {
         XCTAssertEqual(confirmPaymentSourceRequest.method, expectedMethod)
         XCTAssertEqual(confirmPaymentSourceRequest.headers, expectedHeaders)
     }
+
+    enum TestError: Error {
+        case encodeError
+    }
+    
+    class FailingJSONEncoder: JSONEncoder {
+        
+        override func encode<T>(_ value: T) throws -> Data where T: Encodable {
+            throw TestError.encodeError
+        }
+    }
```

Comment:
Nitpick - I would move this all into a `FailingJSONEncoder` file in this test module

Reply:
Addressed here: 7f57a8067d0624219df2ff69d2edf49deaf397c5
I put it under Test Shared and imported that module.

Reply:
Awesome! And thank you for linking to the commit, super easy to find and read